web/themes/custom/novel/novel.theme

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Novel Theme.
+ */
+
+/**
+ * Implements template_preprocess_page().
+ *
+ * @param mixed[] $variables
+ *   Theme variables.
+ */
+function novel_preprocess_page(array &$variables): void {
+  $data = [
+    // @todo Something Insert the right search result page url when it is ready.
+    'search-header-url' => '/search',
+    'alt-text' => t('Search field', [], ['context' => 'Search Header']),
+    'input-placeholder' => t('Start typing in order to search', [], ['context' => 'Search Header']),
+  ];
+
+  $variables['search'] = [
+    'header' => dpl_react_render('search-header', $data),
+  ];
+}
+
+/**
+ * Implements hook_page_attachments()
+ *
+ * @param mixed[] $page
+ *   Drupal render array.
+ */
+function novel_page_attachments(array &$page): void {
+  $metatag = [
+    '#tag' => 'meta',
+    '#attributes' => [
+      'name' => 'description',
+      'content' => t('This is the meta description of the DPL CMS site'),
+    ],
+  ];
+
+  $page['#attached']['html_head'][] = [$metatag, 'description'];
+}
```

Comment:
Please remember to remove this. It is currently not needed to improve pagespeed score and in the future we will probably want to address this using the `metatag` module. Also the current implementation does not seem to work 😅 .